src/features/licenses/components/LicenseForm/index.jsx

```diff
@@ -65,6 +65,16 @@ export const LicenseForm = ({
           </PageBanner>
         </Form.Group>
         )}
+      <Form.Group isInvalid={has(errors, 'licenseName')}>
+        <Form.Label>License Name</Form.Label>
+        <Form.Control
+          name="licenseName"
+          maxLength="64"
```

Comment:
Is it necessary to add this? I assume this is not the same max length in the backend. @sergivalero20

Reply:
@Squirrel18 you're right. I just changed the max length.